### ISSUE TITLE
Wasm test failure: System addon update list error SyntaxError: XMLHttpRequest.open

### ISSUE THREAD

**Issue by jkotas** [dotnet/runtime#102955]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=694729
Build error leg or test failing: WasmTestOnChrome-ST-System.Memory.Tests.WorkItemExecution
Pull request: https://github.com/dotnet/runtime/pull/102946

## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssueJsonStepByStep.md).

```json
{
  "ErrorMessage": [ "System addon update list error SyntaxError: XMLHttpRequest.open" ],
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=694729
**Error message validated:** `[System addon update list error SyntaxError: XMLHttpRequest.open`]
**Result validation:** :x: Known issue did not match with the provided build.
**Validation performed at:** 6/1/2024 4:56:55 PM UTC

### Report

|Build|Definition|Test|Pull Request|
|---|---|---|---|
|[712763](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712763)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712763&view=ms.vss-test-web.build-test-results-tab&runId=17824576&resultId=189577)|dotnet/runtime#103635|
|[712742](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712742)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712742&view=ms.vss-test-web.build-test-results-tab&runId=17823878&resultId=189577)|dotnet/runtime#103697|
|[712737](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712737)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712737&view=ms.vss-test-web.build-test-results-tab&runId=17823676&resultId=189406)|dotnet/runtime#103634|
|[712718](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712718)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712718&view=ms.vss-test-web.build-test-results-tab&runId=17822824&resultId=189577)|dotnet/runtime#103412|
|[712690](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712690)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712690&view=ms.vss-test-web.build-test-results-tab&runId=17822028&resultId=189507)|dotnet/runtime#103639|
|[711372](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711372)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711372&view=ms.vss-test-web.build-test-results-tab&runId=17821184&resultId=186718)|dotnet/runtime#103621|
|[712486](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712486)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712486&view=ms.vss-test-web.build-test-results-tab&runId=17819768&resultId=189559)|dotnet/runtime#102403|
|[711428](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711428)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711428&view=ms.vss-test-web.build-test-results-tab&runId=17819128&resultId=188790)|dotnet/runtime#101337|
|[712515](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712515)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712515&view=ms.vss-test-web.build-test-results-tab&runId=17817068&resultId=186722)||
|[706184](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706184)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706184&view=ms.vss-test-web.build-test-results-tab&runId=17816876&resultId=188030)|dotnet/runtime#103186|
|[712064](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712064)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712064&view=ms.vss-test-web.build-test-results-tab&runId=17815788&resultId=189500)|dotnet/runtime#102670|
|[712361](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712361)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Text.RegularExpressions.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=712361&view=ms.vss-test-web.build-test-results-tab&runId=17809818&resultId=191468)|dotnet/runtime#102655|
|[711360](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711360)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711360&view=ms.vss-test-web.build-test-results-tab&runId=17780686&resultId=189512)|dotnet/runtime#102670|
|[711326](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711326)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711326&view=ms.vss-test-web.build-test-results-tab&runId=17778992&resultId=186659)|dotnet/runtime#103422|
|[711308](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711308)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711308&view=ms.vss-test-web.build-test-results-tab&runId=17778238&resultId=186718)||
|[711276](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711276)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711276&view=ms.vss-test-web.build-test-results-tab&runId=17777450&resultId=188048)|dotnet/runtime#103621|
|[711142](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711142)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=711142&view=ms.vss-test-web.build-test-results-tab&runId=17773592&resultId=189531)|dotnet/runtime#103612|
|[710992](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710992)|dotnet/runtime|[System.Text.RegularExpressions.Tests.RegexPcre2Tests.IsMatchTests](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710992&view=ms.vss-test-web.build-test-results-tab&runId=17768776&resultId=147780)|dotnet/runtime#102655|
|[704663](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704663)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704663&view=ms.vss-test-web.build-test-results-tab&runId=17763360&resultId=182406)|dotnet/runtime#103328|
|[710807](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710807)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710807&view=ms.vss-test-web.build-test-results-tab&runId=17761992&resultId=189531)|dotnet/runtime#103599|
|[710799](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710799)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710799&view=ms.vss-test-web.build-test-results-tab&runId=17761608&resultId=189531)||
|[710789](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710789)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710789&view=ms.vss-test-web.build-test-results-tab&runId=17761522&resultId=189531)|dotnet/runtime#101337|
|[710004](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710004)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710004&view=ms.vss-test-web.build-test-results-tab&runId=17759816&resultId=189531)|dotnet/runtime#103422|
|[710580](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710580)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Net.Http.Functional.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710580&view=ms.vss-test-web.build-test-results-tab&runId=17755108&resultId=100290)|dotnet/runtime#103591|
|[710515](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710515)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710515&view=ms.vss-test-web.build-test-results-tab&runId=17754344&resultId=189531)|dotnet/runtime#103339|
|[710167](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710167)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Memory.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710167&view=ms.vss-test-web.build-test-results-tab&runId=17745220&resultId=188600)|dotnet/runtime#103572|
|[710041](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710041)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=710041&view=ms.vss-test-web.build-test-results-tab&runId=17741114&resultId=187718)|dotnet/runtime#103412|
|[709900](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709900)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709900&view=ms.vss-test-web.build-test-results-tab&runId=17735502&resultId=186653)|dotnet/runtime#101337|
|[709769](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709769)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709769&view=ms.vss-test-web.build-test-results-tab&runId=17730490&resultId=186683)|dotnet/runtime#103422|
|[709764](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709764)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709764&view=ms.vss-test-web.build-test-results-tab&runId=17730000&resultId=188609)||
|[709198](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709198)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709198&view=ms.vss-test-web.build-test-results-tab&runId=17728550&resultId=189485)|dotnet/runtime#103529|
|[709579](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709579)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709579&view=ms.vss-test-web.build-test-results-tab&runId=17719648&resultId=180507)||
|[707574](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707574)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707574&view=ms.vss-test-web.build-test-results-tab&runId=17718256&resultId=189439)|dotnet/runtime#103226|
|[709363](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709363)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709363&view=ms.vss-test-web.build-test-results-tab&runId=17714448&resultId=189529)|dotnet/runtime#103412|
|[709260](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709260)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Numerics.Vectors.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709260&view=ms.vss-test-web.build-test-results-tab&runId=17710084&resultId=191435)|dotnet/runtime#103527|
|[709254](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709254)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Net.Http.Functional.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709254&view=ms.vss-test-web.build-test-results-tab&runId=17709414&resultId=100290)|dotnet/runtime#99555|
|[708892](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708892)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708892&view=ms.vss-test-web.build-test-results-tab&runId=17709082&resultId=189517)|dotnet/runtime#103412|
|[708944](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708944)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708944&view=ms.vss-test-web.build-test-results-tab&runId=17708178&resultId=189517)|dotnet/runtime#103416|
|[707989](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707989)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707989&view=ms.vss-test-web.build-test-results-tab&runId=17708170&resultId=188634)|dotnet/runtime#103441|
|[709151](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709151)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Numerics.Vectors.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709151&view=ms.vss-test-web.build-test-results-tab&runId=17706468&resultId=191435)|dotnet/runtime#103527|
|[709132](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709132)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709132&view=ms.vss-test-web.build-test-results-tab&runId=17705730&resultId=189529)||
|[709104](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709104)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Runtime.InteropServices.JavaScript.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=709104&view=ms.vss-test-web.build-test-results-tab&runId=17704320&resultId=100584)|dotnet/runtime#99555|
|[708778](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708778)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708778&view=ms.vss-test-web.build-test-results-tab&runId=17694464&resultId=189517)||
|[708366](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708366)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708366&view=ms.vss-test-web.build-test-results-tab&runId=17684160&resultId=186559)||
|[708273](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708273)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708273&view=ms.vss-test-web.build-test-results-tab&runId=17681774&resultId=189437)|dotnet/runtime#101963|
|[708252](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708252)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Memory.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708252&view=ms.vss-test-web.build-test-results-tab&runId=17681142&resultId=188587)|dotnet/runtime#103308|
|[708232](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708232)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=708232&view=ms.vss-test-web.build-test-results-tab&runId=17680556&resultId=189394)|dotnet/runtime#103487|
|[707934](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707934)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707934&view=ms.vss-test-web.build-test-results-tab&runId=17670886&resultId=186559)|dotnet/runtime#103422|
|[707894](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707894)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707894&view=ms.vss-test-web.build-test-results-tab&runId=17670388&resultId=186602)|dotnet/runtime#103416|
|[707860](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707860)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707860&view=ms.vss-test-web.build-test-results-tab&runId=17669704&resultId=189437)|dotnet/runtime#103473|
|[707826](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707826)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707826&view=ms.vss-test-web.build-test-results-tab&runId=17667808&resultId=189394)|dotnet/runtime#103412|
|[707877](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707877)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Runtime.InteropServices.JavaScript.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707877&view=ms.vss-test-web.build-test-results-tab&runId=17669284&resultId=100712)|dotnet/runtime#103413|
|[707728](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707728)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Memory.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707728&view=ms.vss-test-web.build-test-results-tab&runId=17662388&resultId=188508)|dotnet/runtime#103232|
|[707642](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707642)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707642&view=ms.vss-test-web.build-test-results-tab&runId=17660298&resultId=189437)||
|[707623](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707623)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Numerics.Vectors.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707623&view=ms.vss-test-web.build-test-results-tab&runId=17660000&resultId=191343)|dotnet/runtime#103462|
|[707560](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707560)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Memory.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707560&view=ms.vss-test-web.build-test-results-tab&runId=17657576&resultId=188508)|dotnet/runtime#103454|
|[707164](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707164)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707164&view=ms.vss-test-web.build-test-results-tab&runId=17646010&resultId=185288)|dotnet/runtime#103363|
|[707135](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707135)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707135&view=ms.vss-test-web.build-test-results-tab&runId=17645024&resultId=185287)||
|[707110](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707110)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=707110&view=ms.vss-test-web.build-test-results-tab&runId=17644314&resultId=185287)|dotnet/runtime#103441|
|[706820](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706820)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706820&view=ms.vss-test-web.build-test-results-tab&runId=17635638&resultId=185287)|dotnet/runtime#103422|
|[706851](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706851)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Runtime.InteropServices.JavaScript.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706851&view=ms.vss-test-web.build-test-results-tab&runId=17636238&resultId=100584)|dotnet/runtime#103423|
|[706355](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706355)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706355&view=ms.vss-test-web.build-test-results-tab&runId=17635356&resultId=185282)|dotnet/runtime#103226|
|[698641](https://dev.azure.com/dnceng-public/public/_build/results?buildId=698641)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=698641&view=ms.vss-test-web.build-test-results-tab&runId=17635588&resultId=182363)|dotnet/runtime#103091|
|[706077](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706077)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706077&view=ms.vss-test-web.build-test-results-tab&runId=17634732&resultId=185281)|dotnet/runtime#100056|
|[706770](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706770)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706770&view=ms.vss-test-web.build-test-results-tab&runId=17634268&resultId=185286)|dotnet/runtime#103160|
|[706782](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706782)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Net.WebSockets.Client.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706782&view=ms.vss-test-web.build-test-results-tab&runId=17634002&resultId=100553)|dotnet/runtime#103322|
|[706713](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706713)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706713&view=ms.vss-test-web.build-test-results-tab&runId=17632282&resultId=185244)|dotnet/runtime#103402|
|[706680](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706680)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706680&view=ms.vss-test-web.build-test-results-tab&runId=17632058&resultId=185287)|dotnet/runtime#103416|
|[706610](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706610)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706610&view=ms.vss-test-web.build-test-results-tab&runId=17630018&resultId=185287)|dotnet/runtime#103412|
|[698895](https://dev.azure.com/dnceng-public/public/_build/results?buildId=698895)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=698895&view=ms.vss-test-web.build-test-results-tab&runId=17630638&resultId=182379)|dotnet/runtime#103103|
|[706432](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706432)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706432&view=ms.vss-test-web.build-test-results-tab&runId=17624590&resultId=185116)|dotnet/runtime#103398|
|[706393](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706393)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706393&view=ms.vss-test-web.build-test-results-tab&runId=17623020&resultId=185287)|dotnet/runtime#103402|
|[706379](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706379)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Thread.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706379&view=ms.vss-test-web.build-test-results-tab&runId=17622688&resultId=184313)||
|[706310](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706310)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706310&view=ms.vss-test-web.build-test-results-tab&runId=17621126&resultId=185280)|dotnet/runtime#103397|
|[706307](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706307)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706307&view=ms.vss-test-web.build-test-results-tab&runId=17621116&resultId=185280)|dotnet/runtime#103396|
|[703399](https://dev.azure.com/dnceng-public/public/_build/results?buildId=703399)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=703399&view=ms.vss-test-web.build-test-results-tab&runId=17621728&resultId=182406)|dotnet/runtime#103272|
|[706286](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706286)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706286&view=ms.vss-test-web.build-test-results-tab&runId=17620312&resultId=185121)|dotnet/runtime#103395|
|[706238](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706238)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Net.Http.Functional.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706238&view=ms.vss-test-web.build-test-results-tab&runId=17618470&resultId=100290)|dotnet/runtime#103309|
|[706230](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706230)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=706230&view=ms.vss-test-web.build-test-results-tab&runId=17618424&resultId=185280)|dotnet/runtime#103363|
|[704609](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704609)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704609&view=ms.vss-test-web.build-test-results-tab&runId=17618540&resultId=182406)|dotnet/runtime#103325|
|[697314](https://dev.azure.com/dnceng-public/public/_build/results?buildId=697314)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=697314&view=ms.vss-test-web.build-test-results-tab&runId=17606372&resultId=182354)|dotnet/runtime#100918|
|[704042](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704042)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704042&view=ms.vss-test-web.build-test-results-tab&runId=17604750&resultId=182406)|dotnet/runtime#102960|
|[700763](https://dev.azure.com/dnceng-public/public/_build/results?buildId=700763)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=700763&view=ms.vss-test-web.build-test-results-tab&runId=17603012&resultId=182400)|dotnet/runtime#103163|
|[705694](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705694)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Net.Http.Functional.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705694&view=ms.vss-test-web.build-test-results-tab&runId=17601652&resultId=100290)|dotnet/runtime#97589|
|[704278](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704278)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704278&view=ms.vss-test-web.build-test-results-tab&runId=17601946&resultId=187155)|dotnet/runtime#99747|
|[704231](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704231)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704231&view=ms.vss-test-web.build-test-results-tab&runId=17598948&resultId=182406)|dotnet/runtime#103295|
|[705559](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705559)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Threading.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705559&view=ms.vss-test-web.build-test-results-tab&runId=17597594&resultId=185266)|dotnet/runtime#103279|
|[704459](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704459)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704459&view=ms.vss-test-web.build-test-results-tab&runId=17596378&resultId=187155)|dotnet/runtime#103140|
|[705479](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705479)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Runtime.InteropServices.JavaScript.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705479&view=ms.vss-test-web.build-test-results-tab&runId=17594812&resultId=100584)|dotnet/runtime#103241|
|[701926](https://dev.azure.com/dnceng-public/public/_build/results?buildId=701926)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=701926&view=ms.vss-test-web.build-test-results-tab&runId=17594488&resultId=182400)|dotnet/runtime#103214|
|[705232](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705232)|dotnet/runtime|[WasmTestOnFirefox-MT-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705232&view=ms.vss-test-web.build-test-results-tab&runId=17593218&resultId=181396)|dotnet/runtime#103195|
|[705348](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705348)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Memory.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705348&view=ms.vss-test-web.build-test-results-tab&runId=17592138&resultId=187155)|dotnet/runtime#103344|
|[705344](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705344)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705344&view=ms.vss-test-web.build-test-results-tab&runId=17591784&resultId=182406)|dotnet/runtime#102882|
|[704527](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704527)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=704527&view=ms.vss-test-web.build-test-results-tab&runId=17591584&resultId=187155)|dotnet/runtime#103094|
|[705321](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705321)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Memory.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705321&view=ms.vss-test-web.build-test-results-tab&runId=17591382&resultId=187168)|dotnet/runtime#103191|
|[703230](https://dev.azure.com/dnceng-public/public/_build/results?buildId=703230)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=703230&view=ms.vss-test-web.build-test-results-tab&runId=17591234&resultId=187156)|dotnet/runtime#103263|
|[705150](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705150)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705150&view=ms.vss-test-web.build-test-results-tab&runId=17590812&resultId=187163)|dotnet/runtime#103306|
|[705252](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705252)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Memory.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705252&view=ms.vss-test-web.build-test-results-tab&runId=17589244&resultId=187161)|dotnet/runtime#102780|
|[705238](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705238)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705238&view=ms.vss-test-web.build-test-results-tab&runId=17588382&resultId=182408)|dotnet/runtime#103307|
|[705235](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705235)|dotnet/runtime|[WasmTestOnFirefox-ST-System.Private.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=705235&view=ms.vss-test-web.build-test-results-tab&runId=17588362&resultId=182406)|dotnet/runtime#102909|
Displaying 100 of 384 results
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|12|98|384|

**Comment by maraf** [comment]
All of the logs contains `http://%(server)s/dummy-system-addons.xml` which comes with firefox https://github.com/search?q=org%3Amozilla%20dummy-system-addons.xml&type=code

**Comment by carlossanlop** [comment]
My PR hit this failure in the CI with a variant of the message that did not get caught by KnownBuildError. I am going to open another issue so that it can unblock PRs, but I'm mentioning it here so that you close it as duplicate.

**Comment by maraf** [comment]
> All of the logs contains `http://%(server)s/dummy-system-addons.xml` which comes with firefox

This is probably missleading as the tests continue and eventualy fail on timeout

**Comment by lewing** [comment]
Yeah, we're seeing an uptick in timeouts in System.Memory and System.Xml and for firefox I believe both this and https://github.com/dotnet/runtime/issues/103060 are actually timeouts rather than a problem caused by the firefox internal error

**Comment by maraf** [comment]
Increased timeout. Leaving open for few days

**Comment by kg** [comment]
Yeah just FYI I'm pretty sure 'System addon update list error' is a browser chrome error, not a page error. So it should have nothing to do with whether a test fails, and pattern matching against it may suppress real bugs in PRs.

**Comment by jkotas** [comment]
Feel free to update the pattern to a more appropriate one. This was my best guess what's causing the test to fail.

**Comment by lewing** [comment]
closing in favor of https://github.com/dotnet/runtime/issues/103524
